source/assets/javascripts/locastyle/_table.js

```diff
@@ -0,0 +1,104 @@
+var locastyle = locastyle || {};
+
+locastyle.table = (function() {
+  'use strict';
+
+  var config = {
+    selector: {
+      module: '[data-ls-module=table]',
+      mainCheck: '.ls-main-check',
+      selected: '.is-selected'
+    },
+    classes: {
+      selected: 'is-selected'
```

Comment:
@itumoraes o nome dessa está fora do padrão do Locastyle. Normamente usaríamos: ls-selected. Eu gosto da lógica ls-is-selected , usando o "is, no entanto nã aplicamos isso em nenhum lugar desse projeto.